nms/app/packages/magmalte/app/state/lte/SubscriberState.js

```diff
@@ -238,3 +245,76 @@ export function getSubscriberGatewayMap(subscribers: {[string]: subscriber}) {
   });
   return gatewayMap;
 }
+
+export type SubscriberQueryType = {
+  networkId: string,
+  query: ActionQuery,
+  pageSize: number,
+  tokenList: Array<string>,
+  setTokenList: (Array<string>) => void,
+  ctx: SubscriberContextType,
+  subscriberMetrics?: {[string]: Metrics},
+};
+
+export async function handleSubscriberQuery(props: SubscriberQueryType) {
```

Comment:
Let's add some function comments: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
Some comment about what the expected effects of calling this function are will be helpful for later re-reading of this code.

Reply:
I added a function comment. I don't think I also need to add fields definition here : https://github.com/magma/magma/pull/5875#discussion_r607386211